app/scripts/modules/google/src/loadBalancer/configure/internal/listeners/listener.component.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+import * as _ from 'lodash';
+import { GCE_ADDRESS_SELECTOR } from 'google/loadBalancer/configure/common/addressSelector.component';
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.deck.gce.internalLoadBalancer.listener.component', [GCE_ADDRESS_SELECTOR])
+  .component('gceInternalLoadBalancerListener', {
+    bindings: {
+      loadBalancer: '=',
+      subnetOptions: '=',
+      listener: '=',
+      deleteListener: '&',
+      index: '=',
+      application: '=',
```

Comment:
It's better for these to be `<` rather than `=` if you're not going to be mutating the value passed in (I think that's true of all of the bindings except for `listener`.)